docs/user/reporting/index.asciidoc

```diff
@@ -1,125 +1,166 @@
 +
-When the report completes, a notification appears.
+NOTE: When you create a dashboard report that includes a data table or saved search, the PDF includes only the visible data.
 
-. Click **Download report**.
+. If you are creating a PDF report of a workpad, select *Full page layout* to create a PDF without margins that surround the workpad.
 
-NOTE: When you create a dashboard report that includes a data table or saved search, the PDF includes only the visible data.
+. Generate the report.
+
+. When the report generates, a message appears. On the message, click **Download report**.
+
+. To view and manage reports, open the main menu, then click *Stack Management > Reporting*.
 
 [float]
-[[reporting-layout-sizing]]
-== Layout and sizing
-The layout and size of the PDF or PNG image depends on the {kib} app
-with which the Reporting plugin is integrated. For *Canvas*, the
-worksheet dimensions determine the size for reports. In other apps,
-the dimensions are taken on the fly by looking at
-the size of the visualization elements or panels on the page.
-
-The size dimensions are part of the reporting job parameters. Therefore, to
-make the report output larger or smaller, you can change the size of the browser.
-This resizes the shareable container before generating the
-report, so the desired dimensions are passed in the job parameters.
-
-In the following {kib} dashboard, the shareable container is highlighted.
-The shareable container is captured when you click
-*Generate* or *Copy POST URL* from the *Share* menu. It might take some trial and error
-before you're satisfied with the layout and dimensions in the
-PNG or PDF image.
+[[share-a-direct-link]]
+== Share a direct link
 
-[role="screenshot"]
-image::user/reporting/images/shareable-container.png["Shareable Container"]
+Share a direct link to a saved search, dashboard, or visualization. To access the shared object, authentication is required.
 
+. Open the main menu, then open the saved search, dashboard, or visualization you want to share.
 
+. From the toolbar, click *Share*, then select *Permalinks*.
 
-[float]
-[[optimize-pdf]]
-== Optimize PDF for print&mdash;dashboard only
+. Specify how you want to generate the link:
+
+* To display only the current state of the object, select *Snapshot*.
 
-To create a printer-friendly PDF with multiple A4 portrait pages and two visualizations per page, turn on *Optimize for printing*.
+* To display up-to-date changes, select *Saved object*.
 
+* To generate a shortened link, select *Short URL*.
+
+* To automatically log in anonymous users when you have multiple authentication providers enabled, select *Public URL*.
++
 [role="screenshot"]
-image::user/reporting/images/preserve-layout-switch.png["Share"]
+image::images/permalink-public-url.png[Permalink share menu with Public URL option highlighted]
```

Comment:
This image is blurry.

Reply:
I don't think I see what you see. This looks clear to me.

![Screen Shot 2021-06-30 at 11 23 23 AM](https://user-images.githubusercontent.com/38990715/123997220-c564bc80-d995-11eb-84e1-2708ecd01276.png)